### DISCUSSION TITLE
Import dataset YAML in GUI

### DISCUSSION THREAD

**Discussion by omnidepp** [apache/superset#18442] [Ideas]
**Is your feature request related to a problem? Please describe.**
I can happily export/download complex dataset-configurations through the GUI as a YAML files:
![Bildschirmfoto 2021-02-11 um 16 50 57](https://user-images.githubusercontent.com/14217074/107661280-6ffb6d80-6c89-11eb-9184-3a2397e415d2.png)

 However, I cannot import these configuration in a similar manner. 

**Describe the solution you'd like**
I would like to upload/import dataset yaml files through the GUI.

**Describe alternatives you've considered**
I'm using a docker setup, so using the commandline tool requires significantly more effort like additional volumes etc.

**Comment by Deninc** [comment]
+1, this is a very confusing UI to have an export button but not an import one.
If the feature is not completed then it's better to just disable both.

**Comment by giovannipapini-agilelab** [comment]
Are there any updates regarding this?
